easybuild/easyconfigs/q/QuantumESPRESSO/want-2.5.1-gfortran.patch

```diff
@@ -0,0 +1,14 @@
+For backwards compatability gfortran has an intrinsic iargc so need to use that
```

Comment:
@ocaisa please include author or ref where you got this patch, and fix typo ('`compatibility`')